lte/gateway/python/integ_tests/s1aptests/test_services_are_running.py

```diff
@@ -70,33 +69,42 @@ def test_services_are_running(self):
         for service, state in failed_services.items():
             print(f'{service} failed with reason(s) "{state[SERVICE_RESULT]}"')
 
-        assert not failed_services, "Services are not running correctly. See logging above."
-
-    def _get_services(self):
-        """
-        Returns a list of all services managed by magmad and additionally magmad itself,
-        openvswitch-switch and sctpd.
-        """
-        non_magmad_services = [
-            'magma@magmad',
-            'openvswitch-switch',
-            'sctpd',
-        ]
-
-        raw_magmad_yml = self._s1ap_wrapper.magmad_util.exec_command_output('cat /etc/magma/magmad.yml')
-        magmad_yml = yaml.load(raw_magmad_yml, Loader=SafeLoader)
-        magmad_services = magmad_yml['magma_services']
-
-        return non_magmad_services + [f'magma@{service}' for service in magmad_services]
+        assert not failed_services, \
+            "Services are not running correctly. See logging above."
 
     def _query_state_of_services(self, service_status):
         for service, status in service_status.items():
-            active_state = self._s1ap_wrapper.magmad_util.check_service_activity(
-                f'systemctl is-active {service}',
-            ).strip()
-            start_time = self._s1ap_wrapper.magmad_util.exec_command_output(
-                f'systemctl show {service} --property=ActiveEnterTimestamp',
-            ).strip()
+            if self._s1ap_wrapper.magmad_util.is_service_active(service):
+                active_state = "active"
+            else:
+                active_state = "not active"
+
+            service_name = (
+                self._s1ap_wrapper.magmad_util.
+                get_service_name_from_init_system(service)
+            )
+            if (
+                self._s1ap_wrapper.magmad_util.init_system.value
+                == InitMode.SYSTEMD.value
+            ):
+                start_time = (
+                    self._s1ap_wrapper.magmad_util.exec_command_output(
+                        f'systemctl show {service_name} '
+                        f'--property=ActiveEnterTimestamp',
+                    ).strip()
+                )
+
+            elif (
+                self._s1ap_wrapper.magmad_util.init_system.value
+                == InitMode.DOCKER.value
+            ):
+
```

Comment:
nit³: newline not consistent with above

Reply:
I would disagree, since I would see a difference between a binary operator `==` (line change before) and a variable assignment `=` (parentheses and line change afterwards) in terms of [line change / python style](https://peps.python.org/pep-0008/#should-a-line-break-before-or-after-a-binary-operator). Or am I missing your point?

Reply:
Line 101

Reply:
Ah, then I misunderstood. Yeah, that's not pretty. Changed.